### ISSUE TITLE
❌ Invalid Report: Shop

### ISSUE THREAD

**Issue by ku6039510** [elementor/elementor#20076]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Sir You Can Resolve The Elemetor Issue

### Steps to reproduce

Please

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

Sir Please

**Comment by nicholaszein** [comment]
Hey @ku6039510, 

Thank you for submitting your issue. 🙏 

While reviewing your report, we noticed it was missing some key pieces of information.

👉 Please follow the guidelines for opening a [new issue report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

⚠️ This report is missing essential information: 

* **Clear and concise title:** You did not write a clear and concise title;
* **Prerequisites:** please search the repository before submitting an issue;
* **Description:** you did not provide any description of your issue;
* **Steps to replicate:** You did not provide any way for us to try to reproduce your issue;
* **System info:** You did not provide any information about your system info.

Without them, we simply can't help. 😕 

⚠️ Note that GitHub issues are reserved for 🐞**Bug Reports**, and should not be used for support.

If you have questions or need general support, please use one of the following channels:

* [Elementor Community](https://facebook.com/groups/Elementors/) on Facebook
* [Elementor Knowledge base](https://docs.elementor.com)
* [Support forums](https://wordpress.org/support/plugin/elementor) on WordPress.org 
* [Elementor Developers docs](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Best regards
